vendor/github.com/Shopify/sarama/.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
 - 1.4.3
-- 1.5.2
```

Comment:
Honestly, I have no idea how I managed to pull in a newer version of this dep. Do you think its worth fixing?